src/adapters/DebugProtocolAdapter.ts

```diff
@@ -49,6 +58,7 @@ export class DebugProtocolAdapter {
      * @param handler
      */
     public on(eventName: 'cannot-continue', handler: () => void);
+    public on(eventname: 'chanperf-event', handler: (output: ChanperfHistory) => void);
```

Comment:
We don't need the word "event" in this....it's already an event name. So the event can just be `chanperf`.

Reply:
Did this to match the existing `rendezvous-event` from the `RendezvousTracker`